main.go

```diff
@@ -71,13 +83,14 @@ func retrievePassengerStats() (io.Reader, error) {
 		fmt.Println(err)
 		return nil, err
 	}
+
 	return bytes.NewReader(out), nil
 }
 
 func parsePassengerXML(xmlData *io.Reader) (passengerStatus, error) {
 	var ParsedPassengerXML passengerStatus
 	dec := xml.NewDecoder(*xmlData)
-	dec.CharsetReader = charset.NewReader
+	dec.CharsetReader = charset.NewReaderLabel
```

Comment:
This is the hard-to-find solution to the charset decoder. There are a couple mentions on golang-nuts and stackoverflow on this.